github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-presubmits.yaml

```diff
@@ -695,6 +695,192 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - name: pull-common-instancetypes-kubevirt-debian-11
+    branches:
+      - main
+    always_run: false
+    run_if_changed: "preferences/base/.*|preferences/components/.*|preferences/debian/.*|tests/functests/.*|tests/vendor/.*|tests/go.*"
```

Comment:
Can you update the `run_if_changed`s to include `preferences/linux/.*` or `preferences/windows/.*` depending on the preference?

Reply:
Why would `preferences/windows/.*` be required in the context of Debian functional tests or do you mean in a follow up?

Reply:
To be extra sure :) sorry, I confused that with the new linux preference base we introduced.

Reply:
Do you still want me to add, at least in these test lines I've added the `preferences/linux/.*`?

Reply:
Done.